src/enums/roleTypes.ts

```diff
@@ -1,9 +1,21 @@
 export enum RoleTypes {
   APPLICANT = 'Applicant',
   COMPLETING_PARTY = 'Completing Party',
+  CUSTODIAN = 'Custodian',
   DIRECTOR = 'Director',
   INCORPORATOR = 'Incorporator',
+  LIQUIDATOR = 'Liquidator',
   PARTNER = 'Partner',
   PROPRIETOR = 'Proprietor',
+  OFFICER = 'Officer',
+  CEO = 'Ceo',
+  CFO = 'Cfo',
+  PRESIDENT = 'President',
+  VICE_PRESIDENT = 'Vice President',
+  CHAIR = 'Chair',
+  TREASURER = 'Treasurer',
+  SECRETARY = 'Secretary',
+  ASSISTANT_SECRETARY = 'Assistant Secretary',
+  OTHER = 'Other',
   SUBSCRIBER = 'Subscriber'
 }
```

Comment:
If this list gets used in other UIs, let's consider moving this to shared enums (or common layer for Nuxt UIs).

Reply:
Nothing to do at the moment. Resolving.